tests/rptest/tests/cluster_config_test.py

```diff
@@ -1409,3 +1413,126 @@ def test_no_kafka(self):
                       password=password,
                       sasl_mechanism="SCRAM-SHA-256")
         rpk.create_topic("testtopic")
+
+
+class ClusterConfigAzureSharedKey(RedpandaTest):
+    segment_size = 1024 * 1024
+    topics = (TopicSpec(
+        partition_count=1,
+        replication_factor=3,
+    ), )
+
+    def __init__(self, test_context):
+        self.si_settings = SISettings(
+            test_context,
+            log_segment_size=self.segment_size,
+            cloud_storage_segment_max_upload_interval_sec=1)
+        super().__init__(test_context,
+                         log_level="trace",
+                         si_settings=self.si_settings,
+                         extra_rp_conf={})
+
+        self.kafka_cli = KafkaCliTools(self.redpanda)
+
+    def get_cloud_log_size(self):
+        s3_snapshot = S3Snapshot(self.topics,
+                                 self.redpanda.cloud_storage_client,
+                                 self.si_settings.cloud_storage_bucket,
+                                 self.logger)
+
+        if s3_snapshot.is_ntp_in_manifest(self.topic, 0) == False:
```

Comment:
nit: perhaps we should follow pep-8 in this boolean check `if not s3_snapshot.is_ntp_in_manifest(self.topic, 0):`